### ISSUE TITLE
an issue about shuffling BN

### ISSUE THREAD

**Issue by DerrickWang005** [CoinCheung/denseCL#1]
In your moco.py,  I think 'dense_k' and 'feat_k' also need to be unshuffled back. Is that right?

**Comment by CoinCheung** [comment]
Yes, it is my fault. I am fixing it and doing experiements to verify it. It might require a couple of days since I only have 8 t4 gpus.

Thanks for telling me this !!! Please tell me if you see more mistakes :)

**Comment by DerrickWang005** [comment]
I'm running that on 8 V100. The moco loss and dense loss can be converged well. However, I find a weird thing. The top1 acc of dense logits can achieve 100% in the second epoch😂. I am not sure that's a good thing.

**Comment by CoinCheung** [comment]
I have fixed the unshuffle problem and did a transfer learning experiment. Thanks again for telling me about this problem, and I think we can close this issue now.
